policyengine_us/variables/gov/states/co/tax/income/credits/income_qualified_senior_housing/co_income_qualified_senior_housing_eligible.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class co_income_qualified_senior_housing_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for Colorado Income Qualified Senior Housing Income Tax Credit"
+    definition_period = YEAR
+    reference = (
+        "https://tax.colorado.gov/income-qualified-senior-housing-income-tax-credit",
+        "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=17",
+    )
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.income_qualified_senior_housing
+
+        age_head = tax_unit("age_head", period)
```

Comment:
senior is also ineligible if they have claimed the property tax exemption under section 39-3-203  (point (2) (b) (III)

Reply:
need to add this clause